src/components/NavButton/index.jsx

```diff
@@ -25,7 +25,13 @@ const GordonNavButton = ({
 }) => {
   const link =
     unavailable !== null || linkPath === null ? (
-      <ListItem divider={divider} button onClick={onLinkClick} disabled={unavailable !== null}>
+      <ListItem
+        divider={divider}
+        button
+        onClick={onLinkClick}
+        disabled={unavailable !== null}
+        className="gc360_link"
```

Comment:
What was the reasoning behind applying the link styles to these? I'm not against the change necessarily, but I think it's outside the scope of this PR - this is a change to the expected styles of an element, whereas this PR should mostly be pure refactoring with on actual changes to the page.

Reply:
Well, we tested with dark mode setting silly colors just to see what isn't referencing it right, and the cards are set to a default dark gray, rather than the yellow we had set in our dark mode for testing... so, it didn't work...

Reply:
Because of that, since this PR I've made changes to a bunch of cards like that, which will be included in future commits for the above reasons, unless there's something I'm missing

Reply:
Ok, I see what you mean now. In one sense, the current code is correct because these aren't all links. But in a more pragmatic sense it probably does make sense to style them all consistently.